### ISSUE TITLE
Error when trying to use it

### ISSUE THREAD

**Issue by mikeymckay** [twcloud/tw5-dropbox#4]
Please see screenshot - there is no way to proceed.

![image](https://user-images.githubusercontent.com/12015/34391761-cd1c56d8-eb58-11e7-8385-1df701dbc454.png)

**Comment by Arlen22** [comment]
Please verify that you are able to access https://dropbox.com/ from another tab in the same browser. Also, I assume you are aware that you are using Apps folder access. You should find the folder in your Dropbox at `/Apps/TW5 in the Sky (Arlen22)`.

**Comment by mikeymckay** [comment]
Correct - I can access dropbox in another tab. I am trying to use the Apps folder as I am not comfortable granting access to my entire dropbox.

Here is the error message from the console:

![image](https://user-images.githubusercontent.com/12015/34421460-6bde734c-ec20-11e7-90e6-e10883489808.png)

**Comment by ghost** [comment]
I am having similar issues, but I am receiving a different errors:

> Failed to load resource: the server responded with a status of 503 (Service Unavailable) 
> https://api.dropboxapi.com/2/users/get_current_account
> 
> Failed to load https://api.dropboxapi.com/2/users/get_current_account: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://twcloud.github.io' is therefore not allowed access. The response had HTTP status code 503.

**Comment by Arlen22** [comment]
Strange. It is working fine for me right now. What browser and OS are you using?

Also, you can try the other version at https://twcloud.github.io/dropbox/

**Comment by ghost** [comment]
After some more troubleshooting it appears as though I was behind a
firewall that was preventing the app from working.

-Marcus

**Comment by kalmir** [comment]
Hi, I see I can access your awesome TW-inside-Dropbox syncing and editing solution via 2 different addresses:

1. [twcloud.github.io/dropbox](https://twcloud.github.io/dropbox/)
2. [twcloud.github.io/tw5-dropbox](https://twcloud.github.io/tw5-dropbox/)

I used no. 1 for a while as this is the version linked from tiddlywiki.com. It worked as it should for a while (until recently), i.e. with automatic saving of edits, even if logging in took a bit of time. Now when I try to log in, I don't get over "Reading HTML file" message phase.

Then I discovered no. 2 here at Github. It is fast which is great. **But for some reason it does NOT automatically save changes to the wiki.** If I want changes to save I have to manually click the button in TW.

I was just wondering if it is possible to make no. 2 to save changes as well? That would be great. If you find time to do it (I have no idea how long it can take), I will send you a donation. Your solution is the ideal for me and I would like to use it long-term, if only I can rely on the save. And thanks for the work done already.

**Comment by Arlen22** [comment]
@kalmir, that turned out to be a fairly easy request. I have not gotten any requests to remove the autosave feature from the other one, so I added it to this one as well. See if autosave works now.

**Comment by kalmir** [comment]
Yes, it works well, thank you lots! I'll suggest in TW5 Google Group the change of the link at tiddlywiki.com.

**Comment by Arlen** [comment]
Hi, I think you've been mentioning the wrong Arlen.

**Comment by kamilchm** [comment]
Hi, I have the same problem as @mikeymckay. I can't proceed when I try to use "App Folder Access".
I see an 404 in the network tab for https://twcloud.github.io/tw5-dropbox/undefined

**Comment by Arlen** [comment]
Wrong Arlen.
